.github/workflows/deploy.yml

```
@@ -0,0 +1,145 @@
+        run: |
+          ssh ${{ secrets.EC2_USER }}@${{ secrets.EC2_HOST }} "
+            cd ${{ env.APP_DIR }}
+            git fetch origin master
+            git reset --hard origin/master
+          "
+
+      - name: Create .env file on EC2
+        run: |
+          ssh ${{ secrets.EC2_USER }}@${{ secrets.EC2_HOST }} "
+            cat > ${{ env.APP_DIR }}/.env << 'ENVEOF'
+      DB_HOST=postgres
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)